orbit/pkg/execuser/execuser_linux.go

```diff
@@ -19,23 +23,44 @@ func run(path string, opts eopts) error {
 		return fmt.Errorf("get user: %w", err)
 	}
 
+	// TODO(lucas): Default to display :0 if user DISPLAY environment variable
+	// could not be found, revisit when working on multi-user/multi-session support.
+	// This assumes there's only one desktop session and belongs to the
+	// user returned in `getLoginUID'.
+	defaultDisplay := ":0"
+
 	log.Info().
 		Str("user", user.name).
 		Int64("id", user.id).
-		Msg("running sudo")
+		Msg("attempting to get user's DISPLAY")
 
-	// Flag `-i` is needed to run the command with the user's context, from `man sudo`:
-	// "The command is run with an environment similar to the one a user would receive at log in"
-	arg := []string{"-i", "-u", user.name, "-H"}
-	for _, nv := range opts.env {
-		arg = append(arg, fmt.Sprintf("%s=%s", nv[0], nv[1]))
+	display, err := getUserDisplay(user.name, opts)
+	if err != nil {
+		log.Error().
+			Str("user", user.name).
+			Int64("id", user.id).
+			Err(err).
+			Msgf("failed to get user's DISPLAY, using default %s", defaultDisplay)
+		display = defaultDisplay
+	} else if display == "" {
+		log.Warn().
```

Comment:
Nit.
Running `who` on my Linux VM doesn't show display number:
```
victor  tty2      2024-04-08 10:12 (tty2)
```
I'm wondering whether this should be Info instead.

Reply:
Could you run 
```
$ loginctl
SESSION  UID USER SEAT  TTY 
      2 1000 luk  seat0 tty2
     c2 1000 luk        

$ loginctl show-session 2 -p Type
```
and let me know the output, my guess is your VM is running Wayland (which we should support but currently we don't).

Reply:
Shared via DM (It was a Wayland session.)